src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -3707,4 +3707,10 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="PreferencesViewIncludeTimestampExportPathTooltip" xml:space="preserve">
     <value>When toggled on, file names of exported images include date and time of export.</value>
   </data>
+  <data name="ElementBindingWarningMessage" xml:space="preserve">
+    <value>A Save As command will treat the workspace as a completely new file and cause element binding to break. Choose Save to preserve element binding with the host document.</value>
```

Comment:
perhaps we could make this message bit less scary - something like:

`A Save As command will create a workspace which is treated as completely new file by Dynamo and existing element binding data will be lost. New element binding data will created as normal. Use Save to preserve element binding with the host document.`